src/utils/retrieveCredentialsInfo.js

```diff
@@ -0,0 +1,67 @@
+import i18next from 'i18next';
```

Comment:
Hmm, I think we cant have it in the utils globally (as I told you earlier on the workshop), since then we wont be able to remove `lmio` part of asab-webui-compontns-lib. It really have to be made twice (I know - code duplicity, but its only for good of the library separation)

Reply:
also, rename it. retrieveCredentialsInfo does not make much sense

Reply:
btw. this file should be placed in src/components/Authz/ and src/lmio/components/User/